packages/util/src/__specs__/menu-keyup-listener.spec.js

```diff
@@ -68,12 +65,12 @@ describe('useMenuKeyEvents()', () => {
     let focused = getByTestId('root-list-item-0')
 
     // ArrowUp
-    fireEvent.keyDown(focused, { key: 'ArrowUp' })
+    fireEvent.keyUp(focused, { key: 'ArrowUp' })
```

Comment:
I went with keyup because on rightArrow  I need the keydown to toggle attribute because ie11 does not support focus-within. I could however make them all keydown but have a special synthetic even callback event on Menu for rightArrow for the rightArrow. What do you think? Require a small mod.

Reply:
Scratch that need to rework close logic to make that work.

Reply:
if it's not a big change i think it would be good just to keep the event similar to what they were...but i'll leave that decision up to you